src/widget/wtrackproperty.cpp

```diff
@@ -1,18 +1,44 @@
+#include "widget/wtrackproperty.h"
+
+// Qt includes
+#include <QMenu>
+#include <QtCore/QItemSelectionModel>
+#include <QtWidgets/QInputDialog>
+#include <QtWidgets/QCheckBox>
+#include <QtWidgets/QWidgetAction>
 
-#include <QDebug>
-#include <QUrl>
+// std includes
+#include <utility>
 
+// Project includes
```

Comment:
remove this line

Reply:
I am yet to push my local changes with code cleanup which will remove stray debug statements, inconsistent formatting and functor syntax for signals and slots.